# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/nodejs/hotfix-ada-inline.patch
# Copyright (C) 2025 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

--- node-v23.10.0/deps/ada/ada.h.vanilla	2025-03-29 11:15:38.011612774 +0100
+++ node-v23.10.0/deps/ada/ada.h	2025-03-29 11:16:03.878907000 +0100
@@ -269,7 +269,7 @@
 
 #else  // ADA_REGULAR_VISUAL_STUDIO
 
-#define ada_really_inline inline __attribute__((always_inline))
+#define ada_really_inline inline /* __attribute__((always_inline)) */
 #define ada_never_inline inline __attribute__((noinline))
 
 #define ada_unused __attribute__((unused))
